src/Roles/README.md

```diff
@@ -0,0 +1,74 @@
+You have to be [logged in](https://github.com/alekseykuleshov/rocket-chat#login) and have relevant permissions.
+
+### ROLES LISTING
+
+```php
+$listing = \ATDev\RocketChat\Roles\Role::listing();
+
+if (!$listing) {
+
+	// Log the error
+	$error = \ATDev\RocketChat\Roles\Role::getError();
+}
+```
+
+### ROLE SYNC
+```php
+$role = new \ATDev\RocketChat\Roles\Role();
```

Comment:
@cvalex This one should be a static call since we don't have any role to send. Like listing - we don't have anything and want to get a list of roles. Same here.